intro/asherw.md

```diff
@@ -0,0 +1,12 @@
+<p align="center">
+<img src="img/" height="400">
```

Comment:
No Image attached!

Reply:
thanks for the feedback, I figured out how to attached the image